src/test/scala/cromwell/engine/db/slick/TestSlickDatabase.scala

```diff
@@ -30,15 +30,15 @@ class TestSlickDatabase(configPath: String) {
       log.debug("Opening test connection setup for " + configPath)
       Database.forConfig("", databaseConfig)
     } flatMap { database =>
-      database.run(SimpleDBIO(_.connection.isValid(1))) recover {
-        case ex =>
-          log.error("Unable to connect to database under config: " + configPath, ex)
-          false
-      } andThen {
+      database.run(SimpleDBIO(_.connection.isValid(1))) andThen {
         case _ =>
           log.debug("Closing test connection setup for " + configPath)
           database.close()
       }
+    } recover {
+      case ex =>
+        log.warn("Unable to connect to database under config: " + configPath, ex)
```

Comment:
This led to "red herrings" in the logs for those without mysql setup locally. Ideally we all should for regression testing, but I didn't want it to be mandatory. So technically this is a warning.